pkg/oc/lib/newapp/app/componentmatch.go

```diff
@@ -24,10 +25,10 @@ type ComponentMatch struct {
 
 	// The source of the match. Generally only a single source is
 	// available.
-	Image       *imageapi.DockerImage
-	ImageStream *imageapi.ImageStream
+	DockerImage *dockerv10.DockerImage
```

Comment:
There are places in the code where you're using `imageapi.DockerIamge` please fix them to use this type only.

Reply:
Thanks, done